### PR TITLE
[FUSED] Add new LogSoftmax fused op

### PR DESCRIPTION
**Pull request by mbencer**

### CODE REVIEW

File: src/ngraph/op/fused/log_softmax.hpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        public:
+            NGRAPH_API
+            static constexpr NodeTypeInfo type_info{"Selu", 1};
+            const NodeTypeInfo& get_type_info() const override { return type_info; }
+            /// \brief Constructs a LogSoftmax node.
+            ///
+            /// \param data Node that produces the first input tensor
+            /// \param axis Describes the axis of the inputs when coerced to 2D
+            LogSoftmax(const Output<Node>& data, int64_t axis);
+
```
Comment (diyessi) [inline src/ngraph/op/fused/log_softmax.hpp]: Please add a default no-args constructor.

File: src/ngraph/op/fused/log_softmax.hpp [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+namespace ngraph
+{
+    namespace op
+    {
+        /// \brief LogSoftmax operation
+        class LogSoftmax : public ngraph::op::util::FusedOp
+        {
+        public:
+            NGRAPH_API
+            static constexpr NodeTypeInfo type_info{"Selu", 1};
```
Comment (silee2) [inline src/ngraph/op/fused/log_softmax.hpp]: This must be a typo. `Selu` is a different op.

File: src/ngraph/op/fused/log_softmax.cpp [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+{
+    constructor_validate_and_infer_types();
+}
+
+NodeVector op::LogSoftmax::decompose_op() const
+{
+    const auto data = input_value(0);
+    const auto data_shape = data.get_shape();
+
+    std::vector<size_t> axes(data_shape.size() - m_axis);
```
Comment (silee2) [inline src/ngraph/op/fused/log_softmax.cpp]: what if `m_axis` is negative?

File: src/ngraph/op/fused/log_softmax.hpp (hunk 1 above)
Comment (mbencer) [reply src/ngraph/op/fused/log_softmax.hpp]: Done

File: src/ngraph/op/fused/log_softmax.hpp (hunk 2 above)
Comment (mbencer) [reply src/ngraph/op/fused/log_softmax.hpp]: Done

File: src/ngraph/op/fused/log_softmax.cpp (hunk 3 above)
Comment (mbencer) [reply src/ngraph/op/fused/log_softmax.cpp]: According to arrangements I move handling negative axis from `onnx_importer` to `ngraph op`

File: src/ngraph/validation_util.cpp [hunk 4]
```diff
@@ -794,3 +794,27 @@ PartialShape ngraph::infer_slice_shape(const Node* node,
 
     return dim;
 }
+
+std::size_t ngraph::validate_axis(const Node* node, std::int64_t axis, std::int64_t tensor_rank)
```
Comment (postrational) [inline src/ngraph/validation_util.cpp]: Let's rename this function to `normalize_axis` please.

File: src/ngraph/op/fused/log_softmax.cpp [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+{
+    constructor_validate_and_infer_types();
+}
+
+NodeVector op::LogSoftmax::decompose_op() const
+{
+    const auto data = input_value(0);
+    const auto data_shape = data.get_shape();
+
+    auto valid_axis = ngraph::validate_axis(this, m_axis, data_shape.size());
```
Comment (postrational) [inline src/ngraph/op/fused/log_softmax.cpp]: Let's just call this value `axis`, not `valid_axis`.

**Pull request merged**